notebooks/242-freevc/242-freevc.ipynb

```diff
@@ -0,0 +1,811 @@
+{
```

Comment:
<p>I  think it should be moved under </p><p> </p><h1>Convert Modes to OpenVINO Intermediate Representation </h1><p> </p><p>as all of models conversion will be the same, the comments that we will convert models to ONNx and them mo and which tools/methods used for that, should be before we start to speak about specific model</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>